akka-docs-dev/rst/java/migration-guide-1.0-2.x-java.rst

```diff
@@ -420,3 +420,21 @@ And use of query parameters from ``Uri`` that looked like this:
 should be replaced by:
 
 .. includecode:: code/docs/MigrationsJava.java#query-param
+
+SynchronousFileSource and SynchronousFileSink
+============================================
+
+Both have been replaced by `Source.file(…)` and `Sink.file(…)`.
+
+Update procedure
+----------------
+
+Replace `SynchronousFileSource.create(` with `Source.file(`
+
+Replace `SynchronousFileSource` with `Source`.
+
+Replace `SynchronousFileSink.create(` with `Sink.file(`
+
+Replace `SynchronousFileSink.appendTo(f)` with `Sink.file(f, true)`
+
+Replace `SynchronousFileSink` with `Sink`
```

Comment:
if one has used SynchronousFileSink as a type in a method signature or as a val/var type